src/v/kafka/protocol/schemata/describe_configs_response.json

```diff
@@ -59,7 +60,11 @@
             "about": "The synonym value." },
           { "name": "Source", "type": "int8", "versions": "1+",
             "about": "The synonym source." }
-        ]}
+        ]},
+        { "name": "ConfigType", "type": "int8", "versions": "3+", "default": "0", "ignorable": true,
+          "about": "The configuration data type. Type can be one of the following values - BOOLEAN, STRING, INT, SHORT, LONG, DOUBLE, LIST, CLASS, PASSWORD" },
+        { "name": "Documentation", "type": "string", "versions": "3+", "nullableVersions": "0+", "ignorable": true,
+          "about": "The configuration documentation." }
```

Comment:
afaict we don't need to let this documentation field block progress in that it's purely a UI helper for clients. however, it might be convenient to fill it in if all we need to do is provide a small table of strings to look-up.

Reply:
> afaict we don't need to let this documentation field block progress in that it's purely a UI helper for clients. however, it might be convenient to fill it in if all we need to do is provide a small table of strings to look-up.

I don't know if there is a question here; but, that is my understanding as well. The purpose of this field is to provide an explanation for each property in the DescribeConfgs report.